.gha/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ocaml/opam2:ubuntu-18.04-ocaml-4.07
```

Comment:
I think I would move this file into the `.github` directory, same with the `.gha/.gha-ci.sh` file.